### PR TITLE
Added NextJs, flutter, MongoDB courses in Tamil Language

### PR DESCRIPTION
**Pull request by Ratheshan03**
## What does this PR do?
Added new courses for Tamil language
Added new Languages like Next Js and Flutter content

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Ajay-singh1 — CHANGES_REQUESTED** [review]
Fix the linter errors the marker style must be *.

File: courses/free-courses-ta.md [hunk 1]
```diff
@@ -46,8 +48,8 @@
 
 ### <a id="dsa"></a>Data Structures and Algorithms
 
-* [தமிழில் Data Structures and Algorithms](https://youtube.com/playlist?list=PL_UqaR55i1797oG0BL0wtxdPpa_NYNFLz) - 
-CSE Tamila by Eezytutorials
+* [தமிழில் Data Structures and Algorithms](https://youtube.com/playlist?list=PL_UqaR55i1797oG0BL0wtxdPpa_NYNFLz) -
+    CSE Tamila by Eezytutorials
```
Comment (eshellman) [inline courses/free-courses-ta.md:52]: one line

File: courses/free-courses-ta.md [hunk 2]
```diff
@@ -57,6 +59,13 @@ CSE Tamila by Eezytutorials
 * [Express JS Tutorial In Tamil](https://youtube.com/playlist?list=PLtMr2pEysMV6ArKDOGVmjQxjW-RcdxFHE&si=CR4PI0sjOcAUnXut) - VJ Techno Wizard
 
 
+### Flutter
+
+* [Flutter in Tamil](https://www.youtube.com/playlist?list=PLBngtsPyn30HYEkmqIqZwvLl0c8zWqCtx) - LEARN ALL IN TAMIL
```
Comment (eshellman) [inline courses/free-courses-ta.md:64]: no ALL CAPS

File: courses/free-courses-ta.md [hunk 3]
```diff
@@ -57,6 +59,13 @@ CSE Tamila by Eezytutorials
...
+* [Flutter Tamil Tutorial](https://www.youtube.com/playlist?list=PL_hkki6Usyn4krz6wexRj1baFE-TWKVMb) - Techashonline
```
Comment (eshellman) [inline courses/free-courses-ta.md:65]: Don't invent titles! Use the title given by the resource.

File: courses/free-courses-ta.md [hunk 4]
```diff
@@ -94,6 +103,13 @@ CSE Tamila by Eezytutorials
 ### MongoDB
 
 * [MongoDB Course in Tamil](https://www.youtube.com/playlist?list=PL7BQ4lqtgECRiWoThupyKXRQoDuEV2zy5) - JVL Code
+* [MongoDB Tamil Tutorial for Beginners](https://www.youtube.com/playlist?list=PLfD4W8QfMd5DhXKriTHyHjNzNSe_1I7g1) - MaanavaN Learn Code
+
+
+### Next JS
+
+* [Full stack Next JS Project Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-GmZBy4-tPhwwL8AEjW2t8G) - Tamil Coding Wizard
+* [Next JS Beginner series Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-H86pt2nVb14UB0vKtK74qZ) - Tamil Coding Wizard
```
Comment (eshellman) [inline courses/free-courses-ta.md:112]: Don't invent titles! Use the title given by the resource.

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-ta.md (hunk 3 above)
Comment (Ratheshan03) [reply courses/free-courses-ta.md:65]: Can you please explain me which titles are you referring to?

File: courses/free-courses-ta.md (hunk 4 above)
Comment (Ratheshan03) [reply courses/free-courses-ta.md:112]: Can you please explain me which titles are you referring to?

**Comment by eshellman** [comment]
check each of your links and make sure that the title you use is the same as the title on the resource. Except, no ALL CAPS, escape `|` or replace it with `-`, and it's ok if you truncate bits if the title is too long

**Comment by Ratheshan03** [comment]
@eshellman changes done as requested 👍🏼

File: courses/free-courses-ta.md (hunk 4 above)
Comment (eshellman) [reply courses/free-courses-ta.md:110]:
```suggestion
* [[Next JS Tamil \| Next JS beginner Series Tamil \| Next.js Framework Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-H86pt2nVb14UB0vKtK74qZ)](https://www.youtube.com/playlist?list=PLQeZxRj52I-H86pt2nVb14UB0vKtK74qZ) - Tamil Coding Wizard
```

File: courses/free-courses-ta.md [hunk 5]
```diff
@@ -94,6 +102,13 @@ CSE Tamila by Eezytutorials
 ### MongoDB
 
 * [MongoDB Course in Tamil](https://www.youtube.com/playlist?list=PL7BQ4lqtgECRiWoThupyKXRQoDuEV2zy5) - JVL Code
+* [MongoDB Tamil Tutorial for Beginners](https://www.youtube.com/playlist?list=PLfD4W8QfMd5DhXKriTHyHjNzNSe_1I7g1) - MaanavaN Learn Code
+
+
+### Next JS
+
+* [Next JS Beginner series Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-H86pt2nVb14UB0vKtK74qZ) - Tamil Coding Wizard
+* [Next JS Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-GmZBy4-tPhwwL8AEjW2t8G) - Tamil Coding Wizard
```
Comment (eshellman) [inline courses/free-courses-ta.md:111]:
```suggestion
* [React JS \| Tutorial \| Tamil](https://www.youtube.com/playlist?list=PLQeZxRj52I-GmZBy4-tPhwwL8AEjW2t8G) - Tamil Coding Wizard
```

**Pull request merged**